app/src/main/java/org/exoplatform/fragment/PlatformWebViewFragment.java

```diff
@@ -506,15 +507,20 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
       Log.d("shouldOverride", url);
       // For external and short links, broadcast logout event if done
       if (url.contains(LOGOUT_PATH)) {
+        clearWebviewData();
         mListener.onUserJustBeforeSignedOut();
       }
-      if (url.contains(mServer.getShortUrl()) && !super.shouldOverrideUrlLoading(view, request))  {
+      if ((url.contains(mServer.getShortUrl()) && !super.shouldOverrideUrlLoading(view, request))) {
         // url is on the server's domain, keep loading normally
         return false;
-      } else {
-        // url is on an external domain, load in a different fragment
-        mListener.onExternalContentRequested(url);
-        return true;
+      } else{
+        if (url.contains("/saml") || url.contains("//identity.") || url.contains("google") || url.contains("facebook") || url.contains("linkedin")){
```

Comment:
Why there are double slash before identity ?
Could you filter using the domain i.e : www.google.com or www.facebook.com and have a boolean that indicates that we are in the process of login ? to not mix it with opening external links  ?